File: backend/RunEnv/javaTest/src/main/java/com/example/app/Loader.java
package com.example.app;

import java.util.*;
import org.json.JSONArray;
public class Loader extends Solution {

    public Loader() {}

    public JSONArray execute(JSONArray input) {
        try {
            // Convert input to correspond parameter type
            JSONArray numJson = (JSONArray) input.get(0);
            int[] nums = new int[numJson.length()];
            for (int i = 0; i < numJson.length(); i++) {
                nums[i] = (int) numJson.get(i);
            }

            int target = (int) input.get(1);
            int[] res = super.twoSum(nums, target);

            // Format output
            JSONArray output = new JSONArray();
            JSONArray result = new JSONArray();
            for (int i = 0; i < res.length; i++) {
                result.put(res[i]);
            }
            output.put(result);
            return output;
        }
        catch (Exception e) {
            System.out.println(e.toString());
            return null;
        }
    }

}

File: backend/RunEnv/cppTest_copy/Loader.cpp
#include <iostream>
#include <nlohmann/json.hpp>

#include "Solution.cpp"
using namespace std;
using json = nlohmann::json;

class Loader : Solution {
   private:
    /* data */
   public:
    Loader(/* args */);
    ~Loader();
    json execute(json input) {
        json numJson = input.at(0);
        int target = input.at(1);

        vector<int> res = Solution::twoSum(numJson, target);

        json output = json::array();
        output.push_back(res);
        return output;
    }
};

Loader::Loader(/* args */) {
}

Loader::~Loader() {
}
